.github/workflows/ci.yml

```
@@ -0,0 +1,78 @@
+          pip install -e .
+
+      - name: Download FFmpeg
+        run: |
+          python scripts/download_ffmpeg.py
+
+      - name: Build with PyInstaller
+        run: |
+          pyinstaller --clean pyprocessor.spec
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)